scripts/install/opt/seagate/cortx/motr/libexec/m0reportbug-bundler

```diff
@@ -37,13 +37,12 @@ that contains m0reportbug archives
 that contain Motr forensic data
 that can be used for reporting and researching Motr bugs.
 
-Positional arguments:
-  dest-dir    Target directory; defaults to '$DEFAULT_DEST_DIR'.
-  bundle-id   Support bundle ID; defaults to the local host name
-              (`hostname`).
 Optional arguments:
+  -t <dest-dir>    Target directory; defaults to '$DEFAULT_DEST_DIR'.
+  -b <bundle-id>   Support bundle ID; defaults to the local host name
```

Comment:
-b,-t,-c,-s are required flags, not optional ones.
Please check #L40, why these are kept under optional arguments?